components/x-styling-demo/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@financial-times/x-styling-demo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/Button.cjs.js",
+  "browser": "dist/Button.es5.js",
+  "module": "dist/Button.esm.js",
+  "styleMain": "dist/Button.css",
```

Comment:
I did look into this, I found rework supports a `"style"` property. I can't see that it has much traction but both [Bootstrap](https://github.com/twbs/bootstrap/blob/v4-dev/package.json#L74) and [Tachyons](https://github.com/tachyons-css/tachyons/blob/master/package.json#L6) support it